src/views/Start/Start.js

```diff
@@ -210,7 +216,7 @@ function Start() {
                       <div className="col s8">
                         <div className="learn_level_div_middle">
                           <font className="learn_title">
-                            {sel_lang === 'en' ? 'Sentence' : 'வாக்கியம்'}
```

Comment:
why Tamil removed?

Reply:
Because i have already added another line of code to accept language from .ENV file